src/com/aca/ArmineArzumanyan/homeWork_1/repository/GadgetsRepository.java

```diff
@@ -0,0 +1,33 @@
+package com.aca.armineArzumanyan.homeWork_1.repository;
+
+import com.aca.armineArzumanyan.homeWork_1.items.gadget.Gadget;
+
+import java.util.*;
+
+public class GadgetsRepository{ //implements ItemsRepository<Gadget> {
+HashMap<Integer, Gadget> allGadgets;
+
+    public GadgetsRepository() {
+        allGadgets = new HashMap<>();
+    }
+
+    public HashMap<Integer,Gadget> getAll() {
+        return allGadgets;
+    }
+
+
+    public void GetItemsList(HashMap<Integer, Gadget> itemList) {
```

Comment:
don't start method name with uppercase, in addition, getItems name is more convenient here, in addition, when we put get_ prefix in the method I expect some return value but I see here void return value